uefi/section.go

```diff
@@ -111,7 +153,7 @@ type Section struct {
 	fileOrder   int
 
 	// Type specific fields
-	TypeSpecific interface{} `json:",omitempty"`
+	TypeSpecific *TypeSpecificHeader `json:",omitempty"`
 
 	// For EFI_SECTION_USER_INTERFACE
 	Name string `json:",omitempty"`
```

Comment:
We should probably make this a TypeSpecific field as well.

Reply:
Lets do that in another pr. This is really getting too big